### DISCUSSION TITLE
手机打开白屏

### DISCUSSION THREAD

**Discussion by nishibaiyang** [openkraken/kraken#215] [Q&A]
写了一个简单的demo，生成kraken js文件在安卓手机上打开白屏
import { createElement } from 'rax';
import Text from 'rax-text';
import View from 'rax-view';
import styles from './index.module.css';
import Image from 'rax-image';

function Course() {
  return (
    <View>
      <View
        className={styles.bannerContainer}
      >
        <View className={styles.bannerInfo}>
          <Text>热搜榜</Text>
          <Text>一网打尽</Text>
          <Text>最近更新</Text>
        </View>
        <Image
          className={styles.bannerImage}
          alt="bannerImage"
          source={{uri: 'https://gw.alicdn.com/tfs/TB1bBD0zCzqK1RjSZFpXXakSXXa-68-67.png'}}
        />
      </View>
      <View style={{ width: '100%', height: '100px', overflow: 'scroll' }}>
        <p>
          The overflow property specifies whether to clip content or to add scrollbars
          when an element's content is too big to fit in a specified area.
          The overflow property specifies whether to clip content or to add scrollbars
          when an element's content is too big to fit in a specified area.
          The overflow property specifies whether to clip content or to add scrollbars
          when an element's content is too big to fit in a specified area.
          The overflow property specifies whether to clip content or to add scrollbars
          when an element's content is too big to fit in a specified area.
          The overflow property specifies whether to clip content or to add scrollbars
          when an element's content is too big to fit in a specified area.
        </p>
      </View>
    </View>
  );
}

export default Course;

**Comment by andycall** [comment]
格式有点乱，能把错误日志和实现的代码分开写吗？

**Reply by nishibaiyang** [reply]
上面就是一个简单的页面，没有日志，kraken在手机上显示白屏。web端是可以展示的
